engine/src/main/java/org/hibernate/validator/overview.html

```diff
@@ -10,7 +10,7 @@
 </head>
 <body>
 <p>This is the Hibernate Validator API documentation. Hibernate Validator is the reference implementation of
- https://projects.eclipse.org/projects/ee4j.bean-validation[Jakarta Bean Validation 2.0].
+    https://projects.eclipse.org/projects/ee4j.validation[Jakarta Validation 3.1].
```

Comment:
... so it's 3.1 after all?

Reply:
yes ! 🙈 🙈 🙈 🙈 3.1
I should probably go through it once more to see where the versions are incorrect, as I mostly was looking at the name only in files that are not docs.